pages/post/[post-id]/index.page.tsx

```diff
@@ -54,6 +55,7 @@ const PostPage: NextPage<PageProps> = ({
   canSuggestTags,
 }) => {
   const tagSummary = tags.map((tag) => tag.string).join(" ");
+  dayjs.extend(utc);
```

Comment:
Loading plugins is an expensive operation so you should place this line in `_app.page.tsx`.